utils.js

```diff
@@ -708,7 +716,7 @@ function formatThread(data) {
     muteSettings: data.muteSettings,
     isCanonicalUser: data.is_canonical_user,
     isCanonical: data.is_canonical,
-    canonicalFbid: data.canonical_fbid,
+    canonicalFbid: formatID(data.canonical_fbid),
```

Comment:
@bsansouci The problem with this, is that on `getThreadList`, canonicalFbid is undefined if the thread is not canonical. Calling `toString` on undefined will cause it to throw an exception. Perhaps it would be better to add `toString()` in the `formatID` function?